lib/pbench/cli/server/tree_manage.py

```diff
@@ -61,16 +62,15 @@ def print_tree(tree: CacheManager):
 )
 @click.option(
     "--reclaim-percent",
-    show_default=True,
     is_flag=False,
     flag_value=20.0,
     type=click.FLOAT,
-    help="Reclaim cached data to maintain a target % free space",
+    help="Reclaim cached data to maintain a target % free space [default 20]",
```

Comment:
What was wrong with letting Click show the default?

Reply:
It doesn't. I'm not sure why. So I changed the message and pretending to tell Click to do it seemed rather pointless when I knew it was ignoring me.

Reply:
OK, I wasted too much time on Click documentation, but I *think* I understand. The show default option only works on the `default=` parameter; which in this case is `None` and wouldn't help anyone even if it was actually shown. That is, if you don't specify `--reclaim-percent`, nothing happens. The default display logic doesn't deal with the case where nothing happens if not specified but *if you specify `--reclaim-percent` without a value*, there's a default (`flag_value`).

Reply:
Thanks for the explanation.  I've tangled with Click's documentation before, which I why I decided to ask _you_, instead.  😉